capstone/src/main/java/com/google/sps/servlets/Constants.java

```diff
@@ -47,6 +47,11 @@ public Constants() {}
   public static final String SCHEDULED_DATE_PROP = "scheduled-date";
   public static final String APPLICATION_NAME = "clubhub-step-2020";
   public static final String TIME_ZONE = "America/Los_Angeles";
+  public static final String START_TIME_PROP = "start-time";
+  public static final String END_TIME_PROP = "end-time";
+  public static final String EVENT_TITLE_PROP = "event-title";
+  public static final String EVENT_DESCRIPTION_PROP = "event-description";
+  public static final String TIMEZONE_OFFSET = ":00.000-07:00";
```

Comment:
We should probably stop hard-coding this - there's a cool way to find time zone from the client side, which can be sent over as a parameter (see https://stackoverflow.com/questions/1091372/getting-the-clients-timezone-offset-in-javascript, second answer). Then you can convert whatever time you need to and from epoch time and that makes it all pretty standardized.

Reply:
This is great! I've added something for now but I'll need to wait to test it once Calendar starts working again. Thanks for bringing this up!